pkgs/stdenv/generic/check-meta.nix

```
@@ -86,12 +89,41 @@ let
     allowInsecurePredicate attrs ||
     builtins.getEnv "NIXPKGS_ALLOW_INSECURE" == "1";
 
-  showLicense = license: toString (map (l: l.shortName or "unknown") (lib.lists.toList license));
+  hasSourceProvenance = attrs:
+    attrs ? meta.sourceProvenance;
```

```suggestion
  hasSourceProvenance = attrs: attrs ? meta.sourceProvenance;
```

Why line-break here and not in other places (ie. `isNonSource`)?

---

Because it was a copy-pasta of e.g. `hasLicense`.